recipes/cutlass/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: cutlass
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cutlass/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1938f0e739646370a59ba1f5e365be4c701e8d9e0b9b725d306622e9aafa6b2a
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None"]
```

Comment:
There is no `bld.bat`, but you aren't skipping Windows builds? Do you plan on building for Windows? Once this recipe is placed into a feedstock, Windows CUDA builds will become available, they just aren't used to validate proposed recipes.